buildsystem/python.cmake

```diff
@@ -7,176 +7,160 @@
 # for an example usage, see py/openage/convert/CMakeLists.txt
 
 function(python_init)
-	find_package(Python 3.3 REQUIRED)
+	find_package(Python 3.4 REQUIRED)
```

Comment:
i think this requires a newer (3.0.0) cmake version (in `building.md` and in the `/cmakelists.txt`)

determined by: `git tag --contains=ab6201ab`
Kitware/Cmake@ab6201ab647c13fee44c44822b8502e9848faaaf